src/Protocol/Protocol18x.cpp

```diff
@@ -3336,6 +3336,14 @@ void cProtocol180::WriteMobMetadata(cPacketizer & a_Pkt, const cMonster & a_Mob)
 			break;
 		}  // case mtBat
 
+		case mtChicken:
+		{
+			auto & Chicken = reinterpret_cast<const cChicken &>(a_Mob);
+			a_Pkt.WriteBEUInt8(0x0c);
+			a_Pkt.WriteBEInt8(Chicken.GetAge());
```

Comment:
I think this is wrong. the age should be an int (see #2398) and the protocol only specifies a bool-like value, "IsChild"